.editorconfig

```diff
@@ -0,0 +1,26 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
I don't know if the `.editorconfig` file should be added with this commit.
What do you think?

Reply:
As far as I know editor configs usually are excluded from commits in .gitignore because other developer might using another editor anyway

Reply:
If there's an `. editorconfig`  file in this project some code style issues could be prevented before commits were even pushed (only if the editor/IDE supports it, of course).

Reply:
its ok to add .editorconfig